recipes-devtools/python/python-is-python3.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "Convenience package which ships a symlink to point the python interpreter to python3"
```

Comment:
is this actually needed if you don use `meta-python2` layer?

Reply:
It is good to have it because several python scripts inside the WebKit Tools directory still have python as shebang instead of python3. As far as I tested they work fine with python3 as long as you have the symlink.

One example is the kraken benchmark script run_benchmark. It ends calling  this:
`Tools/Scripts/webkitpy/benchmark_runner/data/plans/kraken.plan:    "create_script": ["python", "make-hosted.py"],`

Reply:
ok. understood.